database/models/articleimage.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable space-before-function-paren */
+/* eslint-disable no-unused-vars */
+/* eslint-disable func-names */
+
+module.exports = (sequelize, DataTypes) => {
+  const ArticleImage = sequelize.define('ArticleImage', {
+    imageLink: {
+      allowNull: false,
+      type: DataTypes.STRING
+    },
+    articleId: {
+      allowNull: false,
+      type: DataTypes.INTEGER
+    },
+  }, {});
+  ArticleImage.associate = function(models) {
+    // associations can be defined here
```

Comment:
Hi, please is there any reason why there isn't an association between `ArticleImage` and `Articles` table?

Reply:
He implemented it bro... its defined on the Articles model, do check it out.

Reply:
Okay thanks for pointing that out, I just needed some clarity because I saw a comment in the file